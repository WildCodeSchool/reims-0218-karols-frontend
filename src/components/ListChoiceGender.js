import React from "react"
import ChoiceGender from "./ChoiceGender"

const ListChoiceGender = () => (
  <div>
    <ListChoiceGender />
    <ListChoiceGender />
  </div>
)

export default ListChoiceGender
