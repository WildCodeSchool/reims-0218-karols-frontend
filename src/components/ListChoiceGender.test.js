import React from "react"
import ReactDOM from "react-dom"
import ListChoiceGender from "./ListChoiceGender"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ListChoiceGender />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ListChoiceGender />).toJSON()
  expect(tree).toMatchSnapshot()
})
