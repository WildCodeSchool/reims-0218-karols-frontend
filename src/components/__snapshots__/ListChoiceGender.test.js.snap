// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="text-center mb-3 col-lg-6"
  >
    <div
      className="mb-4"
    >
      <div
        className="card"
      >
        <div
          className="wrapper text-center"
          style={
            Object {
              "backgroundImage": "url(https://images.unsplash.com/photo-1476493279419-b785d41e38d8?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=4920987ab334960a4d39b9db70555ef5&auto=format&fit=crop&w=1050&q=80)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content card-body"
            >
              <h5
                className="city card-title"
                style={
                  Object {
                    "fontWeight": "500",
                    "marginTop": "10px",
                  }
                }
              >
                <p
                  style={
                    Object {
                      "color": "#FFF",
                    }
                  }
                >
                  Femme
                </p>
              </h5>
              <button
                className="btn btn-secondary"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#fff",
                    "display": "block",
                    "fontSize": "12px",
                    "fontWeight": "700",
                    "lineHeight": "1",
                    "margin": "2em auto 1em",
                    "marginBottom": "40px",
                    "padding": "10px 20px",
                    "position": "relative",
                    "textAlign": "center",
                  }
                }
                type={undefined}
              >
                Sélectionner
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="text-center col-lg-6"
  >
    <div
      className="mb-4"
    >
      <div
        className="card"
      >
        <div
          className="wrapper text-center"
          style={
            Object {
              "backgroundImage": "url(https://images.unsplash.com/photo-1480455624313-e29b44bbfde1?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=153fb8f3b75056ae1c47da65bf392afa&auto=format&fit=crop&w=1050&q=80)",
              "backgroundPosition": "center",
              "backgroundRepeat": "no-repeat",
              "backgroundSize": "cover",
            }
          }
        >
          <div
            className="data"
          >
            <div
              className="content card-body"
            >
              <h5
                className="city card-title"
                style={
                  Object {
                    "fontWeight": "500",
                    "marginTop": "10px",
                  }
                }
              >
                <p
                  style={
                    Object {
                      "color": "#FFF",
                    }
                  }
                >
                  Homme
                </p>
              </h5>
              <button
                className="btn btn-secondary"
                onClick={[Function]}
                style={
                  Object {
                    "color": "#fff",
                    "display": "block",
                    "fontSize": "12px",
                    "fontWeight": "700",
                    "lineHeight": "1",
                    "margin": "2em auto 1em",
                    "marginBottom": "40px",
                    "padding": "10px 20px",
                    "position": "relative",
                    "textAlign": "center",
                  }
                }
                type={undefined}
              >
                Sélectionner
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
